core-bundle/tests/DependencyInjection/Security/ContaoLoginFactoryTest.php

```diff
@@ -52,14 +50,7 @@ public function testConfiguresTheContainerServices(): void
         $this->assertSame('contao.security.frontend_user_provider', (string) $arguments['index_0']);
         $this->assertSame('security.user_checker.contao_frontend', (string) $arguments['index_1']);
         $this->assertSame('contao_frontend', $arguments['index_2']);
-
-        $this->assertSame(
-            [
-                'lock_period' => 300,
-                'login_attempts' => 3,
-            ],
-            $arguments[0]
-        );
+        $this->assertSame(['lock_period' => 300, 'login_attempts' => 3], $arguments[0]);
 
         $this->assertTrue($container->hasDefinition('security.authentication.listener.form.contao_frontend'));
```

Comment:
I think this line and the lines below need to be removed now that you have removed the `ContaoLoginFactory::createListener()` method, don't they? Should also fix the unit tests.